### PR TITLE
improved logging

### PR DESCRIPTION
**Pull request by hiddenalpha**

### CODE REVIEW

**Comment by dominik-cnx** [comment]
Propose to change the title of the PR to "extra logging for statistic collection" or similar

File: src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java [hunk 1]
```diff
@@ -116,15 +116,15 @@ public class RedisquesHttpRequestHandler implements Handler<HttpServerRequest> {
     private final QueueStatisticsCollector queueStatisticsCollector;
 
     public static void init(Vertx vertx, RedisquesConfiguration modConfig, QueueStatisticsCollector queueStatisticsCollector) {
-        log.info("Enable http request handler: " + modConfig.getHttpRequestHandlerEnabled());
+        log.info("Enable http request handler: {}", modConfig.getHttpRequestHandlerEnabled());
```
Comment (dominik-cnx) [inline src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java:119]: Maybe we should say here

Http request handler enabled: {}

**Review by mcweba — APPROVED** [review]

File: src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java (hunk 1 above)
Comment (hiddenalpha) [reply src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java:119]: It is not yet enabled. It is about to get enabled :)

Comment (dominik-cnx) [reply src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java:119]: We just seem to output the state information. If we want to say what we are about to do then I propose to write 

Enabl*ing*

to indicate this.

**Pull request merged**